artemis/placeholder_page_analyzer.py

```diff
@@ -0,0 +1,84 @@
+from typing import List
+
+import requests
+from bs4 import BeautifulSoup
+
+from artemis import http_requests
+
+
+class KeywordProvider:
+    def get_keywords(self) -> List[str]:
+        return [
+            "trwa konserwacja",
+            "dostawcą domeny jest",
+            "domena utrzymywana",
+            "domen dostępnych w",
+            "automatycznie wygenerowane przez",
+            "w trakcie wykonywania operacji wystąpił błąd",
+            "wróć do bezpieczeństwa",
+            "utrzymywana jest na serwerach",
+            "parametry wspólne serwerów",
+            "forbidden",
```

Comment:
Some of these keywords are too generic and may match too mch (e.g. "forbidden")

![image](https://github.com/user-attachments/assets/e3b93467-1a61-43ac-b32f-510ad0cfd3d4)

IMO let's take 5-10 most popular website providers and just match their "no website" pages (even matching with html so that we don't match some text inside)

BTW, make this configurable